docs/application/dotnet/guides/alarm/alarms.md

```diff
@@ -56,6 +56,9 @@ To enable your application to use the alarm functionality:
 
 You can set an alarm which, when it expires, either launches an application or sends a notification to the user:
 
+> [!NOTE]
+> Since Tizen 6.0, the period can be one of the values of `Tizen.Applications.AlarmStandardPeriod` the value will be phase-aligned with other period of alarm.
```

Comment:
@kilig 
I have asked a question regarding the second part in my previous comment, https://github.com/Samsung/tizen-docs/pull/1078#discussion_r407322760.
In the latest commit, the second part of the sentence is removed.
Please verify the previous comment and elaborate the second part so that we can rephrase.